client/auth/self-hosted-instructions.jsx

```diff
@@ -32,7 +32,8 @@ const SelfHostedInstructions = ( { onClickClose, translate } ) => (
 				<br />
 				<a href="http://jetpack.me/install/">
 					{ translate( 'Please follow these instructions to install Jetpack' ) }
-				</a>.
+				</a>
+				.
```

Comment:
oh wow. this is kind of ugly. I guess this won't unintentionally add a space since it's React JSX and not HTML…

Reply:
It doesn't add any whitespace in JSX. One needs to do tricks like `{ ' ' }` to insert whitespace at most places in JSX.

The lonely dot is there because the children of `li` don't fit on one line. So the all get their own line. It's not particularly pretty in this case, but I can understand the rationale behind it.